# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ValveTurningActionGoal action_goal
ValveTurningActionResult action_result
ValveTurningActionFeedback action_feedback
